web/modules/custom/joinup_core/src/WorkflowHelper.php

```diff
@@ -79,6 +79,24 @@ public function getAvailableStatesLabels(FieldableEntityInterface $entity, Accou
     return $allowed_states;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getAvailableStates(FieldableEntityInterface $entity, AccountInterface $account = NULL) {
```

Comment:
Same as above. For someone that got down with the state_machine before jumping into this piece of code, they would probably confuse this with the retrieval of getting multiple states from the transitions. In the transitions, the multiple states list is the `from` states, so the source states. The target states in the transition is the `to` state and is only one. If you have worked a lot with state_machine, that would get confusing in your mind as, reading this method name standalone, you know that since it is in the form it probably refers to target states but your mind thinks of source states for some reason.